src/main/services/business-address/businessAddressService.ts

```diff
@@ -0,0 +1,25 @@
+import { Request } from "express";
+import { saveDataInSession } from "../../common/__utils/sessionHelper";
+import { BUSINESS_ADDRESS, BUSINESS_NAME } from "../../common/__utils/constants";
+import { Company } from "../../model/Company";
+import { Address } from "../../model/Address";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { Session } from "@companieshouse/node-session-handler";
+
+export class BusinessAddressService {
+    private readonly sessionKey = "businessAddress";
+
+    public saveToSession (req: Request): void {
```

Comment:
could we please rename this method to something like saveBusinessAddress or something like that. When we start using mongo db then we would not be needing to save things in session.

Reply:
Done